files/default/event_handler/write_chef_error_handler.rb

```diff
@@ -0,0 +1,14 @@
+
+module WriteChefError
+  class WriteChefError < Chef::Handler
+    def report
+      if run_status.failed?
+        error_file = '/var/log/parallelcluster/bootstrap_error_msg'
+        unless File.exist?(error_file)
+          message = "Failed when running chef recipes (If --rollback-on-failure was set to false, more details can be found in /var/log/chef-client.log and /var/log/cloud-init-output.log.):"
```

Comment:
From the message it seems to me that this can be only used by the head node, right? If true let's explicit this limitation somewhere: comment or name of the function.

I'm saying this because chef-client.log is not available in the compute, so in that case the message should be different. We can for example instead point to: https://docs.aws.amazon.com/parallelcluster/latest/ug/troubleshooting-v3.html